instrumentation/aws-lambda-1.0/library/src/test/groovy/io/opentelemetry/instrumentation/awslambda/v1_0/TracingRequestApiGatewayWrapperTest.groovy

```diff
@@ -66,8 +67,8 @@ class TracingRequestApiGatewayWrapperTest extends TracingRequestWrapperTestBase
           name("/hello/{param}")
           kind SERVER
           attributes {
-            "$SemanticAttributes.FAAS_ID.key" "arn:aws:lambda:us-east-1:123456789:function:test"
-            "$SemanticAttributes.CLOUD_ACCOUNT_ID.key" "123456789"
+            "$ResourceAttributes.FAAS_ID.key" "arn:aws:lambda:us-east-1:123456789:function:test"
+            "$ResourceAttributes.CLOUD_ACCOUNT_ID.key" "123456789"
```

Comment:
this is confusing that we're using resource attributes for span attributes

Reply:
I think @mateuszrzeszutek had made a spec issue about it, but yeah some (or all?) resource attributes are allowed to be on the span for faas.

Reply:
Yep, there's this issue: https://github.com/open-telemetry/opentelemetry-specification/issues/1261
I've even made a PR for that but it looks like it was moved to "after-ga" because of the overall confusion around Resource API...

Reply:
Don't you mean confusion around 1.0 / GA? :-)